apps/omg_eth/test/support/dev_mining_helper.ex

```diff
@@ -75,4 +75,11 @@ defmodule OMG.Eth.DevMiningHelper do
   end
 
   defp hash(message), do: message |> ExthCrypto.Hash.hash(ExthCrypto.Hash.kec())
+
+  defp get_public_key(private_key) do
```

Comment:
this wrapper is typed three times, while it seems one could implement once in `OMG.Signature` and just call in two of the three places.